ethpm_cli/config.py

```diff
@@ -77,7 +90,15 @@ def get_w3(chain_id: int) -> Web3:
     infura_url = f"{SUPPORTED_CHAIN_IDS[chain_id]}.infura.io"
     headers = build_http_headers()
     infura_url = build_infura_url(infura_url)
-    return Web3(load_provider_from_uri(infura_url, headers))
+    w3 = Web3(load_provider_from_uri(infura_url, headers))
+
+    if private_key:
+        owner_address = Account.from_key(private_key).address
+        signing_middleware = construct_sign_and_send_raw_middleware(private_key)
+        w3.middleware_onion.add(signing_middleware)
+        w3.eth.defaultAccount = to_checksum_address(owner_address)
```

Comment:
This might be a good place to issue a `DEBUG` level logging statement indicating that in-flight signing has been enabled for whatever account address.